### ISSUE TITLE
150% and 200% zoom levels prevent the user from logging in

### ISSUE THREAD

**Issue by Michal-Status** [status-im/status-desktop#13484]
# Bug Report

## Description

Selecting 150% or 200% zoom from the settings requires an app restart. After the restart the application window is cropped in a way that prevents the user from logging in. This can be worked around by going full-screen. But even going full-screen does not solve the issue, as it is impossible to close the "Status is in beta" pop-up. The only way to access the app is to delete the entire Status folder and start from scratch. There is no way to use the application in 150% and 200% zoom mode.

## Steps to reproduce

1. Launch Status
2. Go to settings
3. Go to "Appearance" 
4. Set zoom to 150%/200%
5. Restart the application

#### Expected behavior
It is possible to use the application just like on other zoom levels

#### Actual behavior
It is impossible even to log in to the application

### Additional Information

-Screen: 13,3 inch, 2560x1600 resolution
- Status desktop version: https://ci.infra.status.im/job/status-desktop/job/release/job/release%2F2.27.x/6/
- Operating System: macOS

**Both recordings in 150% zoom**

https://github.com/status-im/status-desktop/assets/155960867/6cdf70cc-58c3-45a5-8d6b-8b11f0f6f09a

https://github.com/status-im/status-desktop/assets/155960867/89e811de-1087-4300-b45a-6de97f5dedd0

**Comment by alexandraB99** [comment]
might be fixed by https://github.com/status-im/status-desktop/issues/12360 ?

**Comment by caybro** [comment]
> might be fixed by #12360 ?

I doubt it given that we have hardcoded spacing/values almost everywhere.

**Comment by caybro** [comment]
TLDR: we effectively scale twice on high res displays and OSs (like macOS here)

Some initial analysis; here's the culprit:
```cpp
void dos_qguiapplication_enable_hdpi(const char *uiScaleFilePath)
{
    QGuiApplication::setAttribute(Qt::AA_EnableHighDpiScaling); // <-- we enable here auto scaling
    QGuiApplication::setHighDpiScaleFactorRoundingPolicy(Qt::HighDpiScaleFactorRoundingPolicy::PassThrough);

    QFile scaleFile(QString::fromUtf8(uiScaleFilePath));
    if (scaleFile.open(QIODevice::ReadOnly)) {
        const auto scale = scaleFile.readAll();
        qputenv("QT_SCALE_FACTOR", scale); // <-- here we apply the additional scaling based on the UI settings (the "Zoom" slider")
    }
}
```

There are 2 solutions:
- do not enable `Qt::AA_EnableHighDpiScaling`, user will have to go with 200% manually on hires displays, otherwise everything will be super small
- remove the "Zoom" slider and let the OS handle the scaling (preferred imo)

**Comment by caybro** [comment]
@igor-sirotin @jrainville @alexandraB99 thoughts? ^^

**Comment by caybro** [comment]
@Michal-Status as a quick workaround, you can remove the `Status/ui-scale` file from your settings directory to be able to log back in

**Comment by caybro** [comment]
> > > remove the "Zoom" slider and let the OS handle the scaling (preferred imo)
> > 
> > 
> > @caybro you mean get rid of the zoom feature? If so, depends if people use it. I know on my laptop screen, the 150% is way too big, that's why I wondered if we could do 125%.
> 
> Yeah the preferred way would be to use your OS's scale feature; Qt (Quick) apps will rescale accordingly without us doing anything
> 
> > Do people no "normal" resolution use that feature?
> > > we effectively scale twice on high res displays and OSs (like macOS here)
> > 
> > 
> > Is there a way to detect if we are high res?
> 
> I'm afraid not; ppl telling me that on macOS, even on a 4k display, they don't have any "visible scaling" setting turned on but I somehow doubt it (I think macOS does a 2x implicitly in that situation)

@jrainville I think we can detect that using [`Screen.devicePixelRatio`](https://doc.qt.io/qt-5/qml-qtquick-window-screen.html#devicePixelRatio-attached-prop) (DPR); we'd basically take the slider float value and divide it by the DPR
